java-test-samples/async-lambda-dynamodb/README.md

```diff
@@ -0,0 +1,45 @@
+# App
+
+This project contains an AWS Lambda maven application with [AWS Java SDK 2.x](https://github.com/aws/aws-sdk-java-v2) dependencies.
+
+## Prerequisites
+- Java 1.8+
```

Comment:
Let's be specific and stick to Java 17 because that is what we support as the latest Java runtime on AWS Lambda.